stable/drupal/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: drupal
-version: 0.4.6
+version: 0.4.7
```

Comment:
@seanknox @sameersbn should that rather be a minor version bump when its dependency `mariadb` had a minor bump?